src/jade/post/manipulate_tally.py

```
@@ -272,6 +272,50 @@ def cumulative_sum(
+    fwhm_frac: float | list[float]
+        FWHM fraction(s) to apply. Default is 0.10 (10%) for all energy bins.
+    """
+    # If fwhm_frac is a single float, convert it to a list with the same length as the tally
+    if isinstance(fwhm_frac, (float, int)):
+        fwhm_frac = [float(fwhm_frac)] * len(tally["Energy"])
+    # If fwhm_frac is a list, check that its length matches the number of rows in the tally
+    elif isinstance(fwhm_frac, list):
+        if len(fwhm_frac) != len(tally["Energy"]):
+            raise ValueError(
+                "Length of fwhm_frac list must match number of rows in tally."
+            )
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)